src/logic/safe/utils/__tests__/safeVersion.test.ts

```diff
@@ -36,4 +40,50 @@ describe('Check safe version', () => {
       expect(hasFeature(FEATURES.SAFE_APPS, '1.1.1')).toBe(true)
     })
   })
+
+  describe('isValidMasterCopy', () => {
```

Comment:
I think we should add some more tests so that we cover [all deployed versions](https://github.com/safe-global/safe-deployments/tree/main/src/assets).

Reply:
Fixed the linter and prettier warnings and added tests for 1.0.0 and 1.2.0!